### ISSUE TITLE
[FEATURE]-Don't subscribe to the whole configuration object

### ISSUE THREAD

**Issue by ngenovese11** [PepperDash/epi-videoCodec-ciscoExtended#14]
**Is your feature request related to a problem? Please describe.**
At startup, we subscribe to the entire status object which is a huge amount of data.  We should figure out what we actually need an only subscribe to those values

**Comment by TrevorPayne** [comment]
I agree - but this ends up being a pretty massive (albeit necessary) refactoring effort that needs to be tested heavily.

I'm not even sure it's the massive status object returning that is the issue - I think the issue is more that it's all deserialized into well over 100 objects every time it comes in - there's a lot of overhead there.

I would rather first swap to more discrete parsing of data (like we've done with new features using JSON Path) we want AND THEN figure out what we aren't using and get more precise with our status registration data.

Additionally, I would probably like to use this opportunity to generate a version of this code that utilizes websockets and move away from the SSH connection.  Will speed up our parsing quite a bit, I imagine, as we can rely on data comeing in as a single large chunk and not have to parse it piecemeal to make sure we have a full JObject.
